styles.css

```diff
@@ -0,0 +1,104 @@
+* {
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    font-size: 15px;
+    margin: 0;
+    padding: 0;
+  }
+  
+  body {
+    background-color: #dee2e6;
+  }
+  
+  h1 {
+    font-size: 30px;
+    text-align: center;
+    margin-bottom: 2%;
+    color: #0d1b2a;
```

Comment:
Hey there! Just and idea 💡 

- You can try to set the font color directly on the global selector, or the body selector. That way you don't need to repeat that property.

Reply:
Thank you! I'll implement it!